packages/react-table/src/components/Table/SortColumn.tsx

```diff
@@ -30,7 +30,12 @@ export const SortColumn: React.FunctionComponent<SortColumnProps> = ({
     SortedByIcon = ArrowsAltVIcon;
   }
   return (
-    <button {...props} className={css(className, styles.tableButton)} onClick={event => onSort && onSort(event)}>
+    <button
+      {...props}
```

Comment:
Would it make sense to let the user change the type via SortColumn props?

Reply:
Done

Reply:
Do we also need to register the new type prop in `SortColumnProps` interface? I see the default value type = 'button', but maybe we need also something like `type?: 'button' | 'submit' | 'reset';` in the interface?